ddtrace/profiling/collector/stack.pyx

```diff
@@ -292,7 +294,7 @@ cdef stack_collect(ignore_profiler, thread_time, max_nframes, interval, wall_tim
     # Do not use `threading.enumerate` to not mess with locking (gevent!)
     thread_id_ignore_list = {
         thread_id
-        for thread_id, thread in ddtrace_threading._active.items()
+        for thread_id, thread in chain(periodic_threads.items(), ddtrace_threading._active.items())
```

Comment:
are these threads even visible to Python?

Reply:
They are not accessible from Python, but when you traverse the list of thread states from the runtime state you will hit them. The profiler currently profiles itself, so not doing this would be a "breaking change" i suppose

Reply:
> They are not accessible from Python, but when you traverse the list of thread states from the runtime state you will hit them.

This is good context that might be worth capturing in a comment somewhere.